tests/ProgramTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Xunit;
+
+namespace Microsoft.CodeAnalysis.Tools.Tests
+{
+    public class ProgramTests
+    {
+        [Fact]
+        public void ExitCodeIsOneWithCheckAndAnyFilesFormatted()
+        {
+            var formatResult = new WorkspaceFormatResult()
+            {
+                FilesFormatted = 1
+            };
+            var exitCode = Program.GetExitCode(formatResult, true);
+
+            Assert.Equal(1, exitCode);
+        }
+
+        [Fact]
+        public void ExitCodeIsZeroWithCheckAndNoFilesFormatted()
+        {
+            var formatResult = new WorkspaceFormatResult()
+            {
+                ExitCode = 42,
+                FilesFormatted = 0
+            };
+            var exitCode = Program.GetExitCode(formatResult, true);

```

Comment:
Use named arguments when passing boolean literals.
```suggestion
            var exitCode = Program.GetExitCode(formatResult, check: true);
```